.gitignore

```diff
@@ -3,6 +3,7 @@
 /log/*.log
 /tmp
 tmp
+/public
```

Comment:
Some things in public we actually want. For instance, the error pages are stored in there. I think `public/system` should be good.